### PR TITLE
Ned consul update

### PR DESCRIPTION
**Pull request by ned1313**
### Solution
Update the Compose with Consul doc and exercise files

### Context
Updated default gateway yaml example
Updated the docker-compose file to use 1.2.4 of the Gloo images
Fixed the envoy YAML config
Fixed the default.yaml config
Updated the variables for the Nomad jobs to use 1.2.4

### CODE REVIEW

File: install/docker-compose-consul/data/gloo-system/default.yaml [hunk 1]
```diff
@@ -1,42 +1,16 @@
...
+metadata:
+  name: default
+  namespace: gloo-system
 refreshRate: 15s
-
-# status will be reported by Gloo as "Accepted"
-# if booted successfully
-status: {}
+vaultSecretSource:
+  address: http://172.17.0.1:8200
```
Comment (ilackarms) [inline install/docker-compose-consul/data/gloo-system/default.yaml]: same as #1890 i'm not sure we can just replace `host.docker.internal` with `172.17.0.1` here, as the latter is likely to change for non-default installation of docker

**Comment by yuval-k** [comment]
HI ned ! can we update to 1.2.5 release? it contains a security fix so i think it would be nice to update our docs to it

File: install/docker-compose-consul/data/gloo-system/default.yaml (hunk 1 above)
Comment (ned1313) [reply install/docker-compose-consul/data/gloo-system/default.yaml]: Updated to use consul and vault.

**Pull request merged**